assets/javascripts/discourse/initializers/category-experts-post-decorator.js

```diff
@@ -31,8 +31,10 @@ function setPostCategoryExpertAttributes(
 }
 
 function initializeWithApi(api) {
-  const requiresApproval = api.container.lookup("site-settings:main")
-    .category_experts_posts_require_approval;
+  const requiresApproval =
+    api.container.lookup(
+      "site-settings:main"
```

Comment:
```suggestion
      "service:site-settings"
```
Let's switch to the new name while we're here